contracts/starknet/space/space.cairo

```diff
@@ -0,0 +1,134 @@
+%lang starknet
+
+from starkware.starknet.common.syscalls import ( get_caller_address, get_block_number )
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.math import assert_lt, assert_le, assert_nn, assert_not_zero
+
+struct Vote:
+    member choice: felt # TODO use Choice enum
+    member eth_address: felt # TODO: use address
+    member voting_power: felt
+end
+
+struct Proposal:
+    member execution_hash: felt # TODO: Use Hash type
+    member start_block: felt
+    member end_block: felt
+end
+
+# TODO: use L1Address instead of felt
+@contract_interface
+namespace IVotingStrategy:
```

Comment:
Put this in a different file (contracts/starknet/strategies/IVotingStrategy.cairo)

Reply:
https://github.com/snapshot-labs/sx-core/pull/47/commits/cfedfb08beec24e956cf59f855a2118e54051567 :D